src/hooks/useLocalStorage.ts

```diff
@@ -0,0 +1,33 @@
+import { useState } from 'react';
+
+interface LocalStorageProps {
+  key: string;
+  initialValue: any; // 바꿔야 됨
+}
+// useLocalStorage : localStorage 관리하는 커스텀 훅
+export function useLocalStorage({ key, initialValue }: LocalStorageProps) {
```

Comment:
```suggestion
interface LocalStorageProps<T> {
  key: string;
  initialValue: T;
}

export function useLocalStorage<T>({ key, initialValue }: LocalStorageProps<T>) {
```

여기서는 호출 시점에 결정되도록 제네릭 타입을 사용하는 것을 추천해봅니다🤩

Reply:
ㅠㅠㅠㅠ감사합니다